Documentation/improvements.md

```diff
@@ -0,0 +1,41 @@
+###This page describes new API features which is available in current development release and does not exist in XNA.
```

Comment:
Don't make this a heading.... just make it regular text.  Then it will look more like this page:

http://www.monogame.net/documentation/?page=Using_The_Pipeline_Tool

With the title, then a paragraph explaining the content of the page, then the first heading.